securedrop/tests/test_source.py

```diff
@@ -2,70 +2,68 @@
 import gzip
 import json
 import re
+import subprocess
 
 from cStringIO import StringIO
-from flask import session, escape, url_for, current_app
-from flask_testing import TestCase
+from flask import session, escape, current_app
 from mock import patch, ANY
 
-from sdconfig import config
+import crypto_util
 import source
 import utils
 import version
 
 from db import db
 from models import Source
+from source_app import main as source_app_main
 from utils.db_helper import new_codename
+from utils.instrument import InstrumentedApp
 
 overly_long_codename = 'a' * (Source.MAX_CODENAME_LEN + 1)
 
 
-class TestSourceApp(TestCase):
+class TestSourceApp:
```

Comment:
We can get rid of the `TestSourceApp` class now, which I think would be nice to flatten this file a bit (if you make this change you could do it in an additional commit if it's easier)

Reply:
I think we need this class for `setup` and `teardown`?

Reply:
We don't use those anymore. Lifecycles are managed by `pytest` alone.